docs/api.rst

```diff
@@ -0,0 +1,47 @@
+API
+===
+This part of the documentation describes the interfaces for using PyPOM.
+
+
+.. py:module:: pypom
+
+.. _WebView:
```

Comment:
I think we should remove `WebView` here and just document `Page` and `Region` (at least for now).